### ISSUE TITLE
Support touch move events of gearvr controller

### ISSUE THREAD

**Issue by rzr** [chenzlabs/gearvr-controls#3]
Hi,

I know this component is merged into aframe:
https://github.com/aframevr/aframe/pull/2183

But is this project still alive for experimentation ? or at least a place for support ?

I have observed  that only touch down and up events are displayed on gearvr model when sVRbrowser is in stereo mode, while in 2D mode the touch moves are well managed.

May this issue not specific to gearvr-controls but maybe samsung internet or anywhere else between browser API and A-Frame.

Hints welcome.

**Comment by machenmusik** [comment]
Question: When you say sVRbrowser do you mean the Samsung Internet (Gear)VR browser, rather than the Oculus Browser, or the normal Samsung Internet browser?

**Comment by rzr** [comment]
yes the SamsungInternet VR browser  (com.sec.android.app.svrbrowser)

I am making a PoC app and sharing notes:
https://github.com/rzr/webthing-iotjs/wiki/XR

**Comment by machenmusik** [comment]
At one point there was some code for the trackpads on the GearVR in the Samsung Internet browser that, when in immersive mode, translated pad swipes into joystick movements instead of exposing as touchpad events.  If that code is still there, that is almost certainly your issue.
